src/ViewLayer/RaceModeDisplay/RaceModeDashboardView.cpp

```diff
@@ -192,23 +197,57 @@ void RaceModeDashboardView::updateFaultLabel(QLabel& dashboardLabel, FaultLabel
     }
 }
 
+void RaceModeDashboardView::initalizeFaultAnimation()
+{
+    fadeEffect_.reset(new QGraphicsColorizeEffect(&(ui_.raceModeDashboard())));
+    fadeEffect_->setColor(QColor(0, 0, 0));
+    ui_.raceModeDashboard().setGraphicsEffect(fadeEffect_.get());
+    faultAnimation_.reset(new QPropertyAnimation(fadeEffect_.get(), "color"));
+    faultAnimation_->setDuration(FAULT_UPDATE_PERIOD / 2 - 50);
+    faultAnimation_->setStartValue(QColor(0, 0, 0));
+    faultAnimation_->setEndValue(QColor(255, 127, 0));
+    connect(faultAnimation_.get(), &QAbstractAnimation::finished, this, &RaceModeDashboardView::reverseFaultAnimation);
+}
+
+void RaceModeDashboardView::runFaultAnimation()
+{
+    numberOfActiveFaults_ = batteryFaultsList_.numberOfActiveLabels() +
```

Comment:
numberOfActiveFaults_ shouldn't need to be a member of RaceModeDashboardView. It just needs to be a local variable in this function as it doesn't need to be used anywhere else.